index.html

```diff
@@ -27,7 +27,7 @@
 
     <link id="favicon" rel="shortcut icon" href="./public/favicon.ico" />
 
-    <link rel="manifest" href="/manifest.json" />
+    <link rel="manifest" href="./manifest.json" />
```

Comment:
relative path for manifest is not working if you open https://1588--pr-cinny.netlify.app/login/matrix.org as it search for it at https://1588--pr-cinny.netlify.app/login/manifest.json

absolute path should be prefixed automatically by vite BASE_URL on build, but seems like it's not happening.

Reply:
>absolute path should be prefixed automatically by vite BASE_URL on build, but seems like it's not happening.

Yeah, that is probably needed now that we have `/login`.
The `pdf.worker.min.js` will probably need BASE_URL or `import.meta.env.BASE_URL` now too I guess